packages/block-editor/src/components/block-list/style.scss

```diff
@@ -104,13 +109,23 @@
 		z-index: 0;
 		pointer-events: none;
 		transition: border-color 0.1s linear, border-style 0.1s linear, box-shadow 0.1s linear;
+		top: -$default-block-margin / 2;
 		right: 0;
 		left: 0;
-		top: -$default-block-margin / 2;
+		border-radius: $radius-block-ui;
```

Comment:
We've removed this radius intentionally in master, we should remove it here too.

Reply:
Ah yep, merge conflict issue! Thanks for spotting, I've now removed that line.